tests/helpers.py

```diff
@@ -73,4 +73,4 @@ def force_leave_swarm(client):
             if e.explanation == "context deadline exceeded":
                 continue
             else:
-                raise
+                return
```

Comment:
It was overzealous - we don't really care if there's an error during the `tearDown`